dcp_prototype/backend/wrangling/migrations/common/metadata.py

```diff
@@ -1,15 +1,9 @@
 """This module contains simple python classes that correspond to the artifact schema"""
 
-import json
-
 
 class MetadataBase(object):
     """Base class for the metadata objects"""
 
-    def to_json(self):
```

Comment:
This function is used in migrate_dcp_one_to_dcp_two and create_artifact

Reply:
I couldn't find any uses of it actually in either file; could you point me to the references please?

Reply:
My bad, I had already refactored that part.  It's not needed.